libsplinter/src/protocol/mod.rs

```diff
@@ -73,6 +73,8 @@ pub(crate) const OAUTH_CALLBACK_MIN: u32 = 1;
 pub(crate) const OAUTH_LOGIN_MIN: u32 = 1;
 #[cfg(all(feature = "oauth", feature = "rest-api-actix"))]
 pub(crate) const OAUTH_LOGOUT_MIN: u32 = 1;
+#[cfg(all(feature = "oauth", feature = "oauth-user-list"))]
```

Comment:
this should include `feature = "rest-api-actix"`

Reply:
Updated this guard to use the `oauth-user-list` and `rest-api-actix` features, as the `oauth-user-list` feature is dependent on `oauth` so this was unnecessary.